### PR TITLE
Provide a Dockerfile to easily render the website

### PR DESCRIPTION
**Pull request by NicolasLM**
Signed-off-by: Nicolas Le Manchet <nicolas.le-manchet@ovh.net>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM ruby:1.9-onbuild 
+
```
Comment (yadutaf) [inline Dockerfile]: Maybe add a MAINTAINER section ?

File: README.md [hunk 2]
```diff
@@ -83,10 +83,25 @@ With markdown syntax.
 
 ## Test your tutorial
 
+### With Docker
```
Comment (yadutaf) [inline README.md]: Maybe add "(recommended)" in the title ?

**Comment by yadutaf** [comment]
I love it ! LGTM

**Comment by NicolasLM** [comment]
Done

**Comment by yadutaf** [comment]
@VincentCasse Go ? Good for me

**Comment by NicolasLM** [comment]
Ping @VincentCasse

**Comment by yadutaf** [comment]
Merged. Thanks!

**Pull request merged**